tests/gtests/internals/test_sdpa.cpp

```diff
@@ -1318,6 +1327,7 @@ GPU_TEST_P(sdpa_test_t, perf) {
     if (mask_ptr) { f16_args[DNNL_ARG_ATTN_MASK] = t.m_mask; }
 
     auto loop_sdpa_f16 = [&] { sdpaf16_p.execute(strm, f16_args); };
+#if 0
```

Comment:
The commented-out reorder_sdpa parts look a bit messy. Is it OK to remove them entirely?